nix/nix/sources.json

```diff
@@ -29,23 +29,17 @@
         "homepage": null,
         "owner": "serokell",
         "repo": "opam-nix",
-        "rev": "bdd7e6730bdf0ea91ada3ebc68387424b087c9f8",
-        "sha256": "1wda717d6391vbgrp0vcv27pxfj4xy1511mssky8ll3iy7i851hn",
+        "rev": "ee285a3b6e05dc274f9ebf59ad64d8a4fded915e",
+        "sha256": "0f3dm834zf7y4c4pp8xr12zl0n1xk0zql3a9m2wh6a5shdcdcwj7",
         "type": "tarball",
-        "url": "https://github.com/serokell/opam-nix/archive/bdd7e6730bdf0ea91ada3ebc68387424b087c9f8.tar.gz",
+        "url": "https://github.com/serokell/opam-nix/archive/ee285a3b6e05dc274f9ebf59ad64d8a4fded915e.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "opam-repository": {
         "branch": "master",
-        "description": "Main public package repository for OPAM, the source package manager of OCaml.",
-        "homepage": "https://opam.ocaml.org",
-        "owner": "ocaml",
-        "repo": "opam-repository",
-        "rev": "3e92793804b7f34fca09fb17e2ebafd6d29d90cf",
-        "sha256": "175fdlydmwn6kb09dyz2pz8z2wdsa2lkfys0yqdgv237m67mk2yh",
-        "type": "tarball",
-        "url": "https://github.com/ocaml/opam-repository/archive/3e92793804b7f34fca09fb17e2ebafd6d29d90cf.tar.gz",
-        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+        "repo": "https://gitlab.com/tezos/opam-repository",
+        "rev": "393349af19bb54e3cb790ac8ef54a72adc71aecf",
+        "type": "git"
```

Comment:
That's a lot of potentially useful properties gone IMO. I think we know the value of all of the ones you removed, so why not bring them back?